presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestAggregationFunction.java

```diff
@@ -38,19 +38,46 @@
 import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
 import static com.facebook.presto.sql.analyzer.TypeSignatureProvider.fromTypeSignatures;
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static java.util.Objects.requireNonNull;
 
 public abstract class AbstractTestAggregationFunction
 {
     protected TypeRegistry typeRegistry;
     protected FunctionManager functionManager;
     protected Session session;
+    protected FeaturesConfig featuresConfig;
+
+    protected AbstractTestAggregationFunction()
+    {
+        this(testSessionBuilder().build());
+    }
+
+    protected AbstractTestAggregationFunction(Session session)
+    {
+        this(session, new FeaturesConfig(), new TypeRegistry());
+    }
+
+    protected AbstractTestAggregationFunction(FeaturesConfig config)
```

Comment:
Didn't check extensively. Is this constructor used anywhere?

Reply:
No, but I added different constructors in case we would like to build one with a given `FeaturesConfig`, just like we do for `AbstractTestFunctions`. I can delete the ones I don't need, but it felt like a simple addition that might be useful in the future for others.

Reply:
Please delete dead code. Code should only be introduced with a purpose, not just in case. 😉